<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyFlatInspectionForm</class>
 <widget class="GUIFormBase" name="PropertyFlatInspectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonGetBack">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>39</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;=</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetGeneratedLibList">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonSetFromAll">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>39</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>=&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetLibList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>381</width>
     <height>191</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Page</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>LibID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>LibName</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ItemType</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonEditLibrary">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ライブラリ編集</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Qtres/Edit.png</normaloff>:/Qtres/Edit.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEditLibFolder">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>分類選択</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Qtres/Class.png</normaloff>:/Qtres/Class.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="labelLibFolderName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ＸＸＸ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGetBackAll">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>39</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>今回使用する設定</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSetFrom">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>39</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>=&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>登録されているライブラリリスト</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="toolButtonMasterShapeArea">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>マスター形状</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonInsoectionArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>検査領域</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonAdaptiveArea">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>形状探索</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonWindowArea">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>窓</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonAdaptiveShape">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>特定形状</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="toolButtonShowMaskMap">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>マップ表示</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
